schema/deploy/data_migration/data_migration_add_general_revision.sql

```diff
@@ -0,0 +1,27 @@
+-- Deploy cif:data_migration/data_migration_add_general_revision to pg
+
+BEGIN;
+begin;
+
+create or replace function cif.add_general_revision()
+  returns void as $$
+declare
+  temp_row record;
+begin
+    alter table cif.project_revision disable trigger _100_committed_changes_are_immutable;
+
+    for temp_row in
+      select id from cif.project_revision
+     loop
+      update cif.project_revision
+      set revision_type='General Revision';
+    end loop;
```

Comment:
> This can be done with a single update statement & the function can be converted to pure SQL (instead of plgpsql)

You mentioned this the other day and I forgot to ask--why is SQL better than plgpsql?